cmd/cue/cmd/help.go

```diff
@@ -351,6 +351,13 @@ The following variables are supported:
                 freebsd   illumos   ios       js (wasm)
                 linux     netbsd    openbsd   plan9
                 solaris   windows
+   arch       architecture identifier of the current system. Valid values:
```

Comment:
Nit: please can we make the padding between columns 2 spaces, like `os`?

Also: which Go version is the reference for this list? The minimum Go version that CUE supports?

Reply:
Formatting changed to double-spacing; the slightly longer lines don't look too bad to my eyes :-)

Commit message and PR updated to reference that the list of architectures was taken from the golang commit that released go1.17. Go1.17 was chosen as

(a) it's the oldest version that (I believe) CUE can be built with, given the version constraint found in  `go.mod` and
(b) the underlying information about supported architectures is still valid in golang's `master` branch, today.